task_01/ex7.py

```diff
@@ -0,0 +1,11 @@
+"""Dictionaries, generator expressions."""
+prices = {
+    'apple': 0.40,
+    'banana': 0.50,
```

Comment:
Just a tip. 
It's ok if some things lead to clarity. Here we can put zeroes after numbers just to show that it's dollars and cents. 
Otherwise, such extra zeroes are redundant, cause `%.2f` this expression in our output will add missed numbers at the end of our string ( 1.2 ->1.20)